ledger/src/blockstore_meta.rs

```diff
@@ -28,8 +28,8 @@ pub struct SlotMeta {
     // True if this slot is full (consumed == last_index + 1) and if every
     // slot that is a parent of this slot is also connected.
     pub is_connected: bool,
-    // List of start indexes for completed data slots
-    pub completed_data_indexes: Vec<u32>,
+    // Shreds indices which are marked data complete.
+    pub completed_data_indexes: BTreeSet<u32>,
```

Comment:
```
The change should be backward compatible because Vec and BTreeSet
are both serialized as seq:
https://github.com/serde-rs/serde/blob/ce0844b9e/serde/src/ser/impls.rs#L207-L208
https://github.com/serde-rs/serde/blob/ce0844b9e/serde/src/ser/impls.rs#L216-L217
```

Cool 😃, can we verify this for sanity?

Reply:
Yes, I did manually verified this as well. something like:
```rust
let mut rng = rand::thread_rng();
let mut xs: Vec<i8> = std::iter::repeat_with(|| rng.gen()).take(100).collect();
let data = bincode::serialize(&xs).unwrap();
let set: BTreeSet<i8> = bincode::deserialize(&data).unwrap();
xs.sort();
assert!(set.into_iter().eq(xs))
```